project/cookiecutter.json

```diff
@@ -85,6 +87,12 @@
       "docker_hub": "Docker Hub",
       "gitlab": "GitLab"
     },
+    "storage": {
+      "__prompt__": "Storage backend",
```

Comment:
It should be made clear that these options are only implemented in the stack, and local development always uses filestorage. Maybe "Storage backend for Docker stack"